public/app/plugins/panel/geomap/editor/BaseLayerEditor.tsx

```diff
@@ -19,5 +19,9 @@ export const BaseLayerEditor: FC<StandardEditorProps<MapLayerOptions, any, Geoma
   onChange,
   context,
 }) => {
+  if (config.geomapDisableCustomBaseLayer) {
+    return <div>The base layer is configured by the server admin.</div>;
+  }
```

Comment:
@eunice98k rather than hiding the config, we show:  
![image](https://user-images.githubusercontent.com/705951/126540473-7f6c1f58-16d7-41be-bdbe-3f175bf079ae.png)